### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
[DEMO LINK](https://andriy-shymkiv.github.io/react_movies-list-add-form/)

### CODE REVIEW

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -1,45 +1,90 @@
...
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImdbId('');
+    setCount((prevCount) => prevCount + 1);
+  };
+
+  const fieldsAreFilled = title.trim().length > 0
+    && imgUrl.trim().length > 0
```
Comment (DanilSerzh) [inline src/components/NewMovie/NewMovie.tsx:39]: i think you can remove the length check

**Review by DanilSerzh — APPROVED** [review]

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,45 +1,90 @@
...
+
+  const handleFormSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    onAdd(createdMovie);
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImdbId('');
+    setCount((prevCount) => prevCount + 1);
```
Comment (denys-danyliuk) [inline src/components/NewMovie/NewMovie.tsx:35]:
```suggestion
    event.preventDefault();
    
    onAdd(createdMovie);
    
    setTitle('');
    setDescription('');
    setImgUrl('');
    setImdbUrl('');
    setImdbId('');
    
    setCount((prevCount) => prevCount + 1);
```

File: src/components/NewMovie/NewMovie.tsx [hunk 3]
```diff
@@ -1,45 +1,90 @@
...
+  };
+
+  const handleFormSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    onAdd(createdMovie);
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImdbId('');
```
Comment (denys-danyliuk) [inline src/components/NewMovie/NewMovie.tsx:34]: create function for clearing the form

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]
show an error if field filled with spaces
Знімок екрана 2022-12-07 о 10 53 32
Знімок екрана 2022-12-07 о 10 45 59

**Pull request closed without merging**